src/decoder/stream.rs

```diff
@@ -447,6 +461,11 @@ impl StreamingDecoder {
         self.info.as_ref()
     }
 
+    /// Set decode config
+    pub fn set_decode_config(&mut self, decode_config: DecodeConfig) {
```

Comment:
```suggestion
    pub(crate) fn set_decode_config(&mut self, decode_config: DecodeConfig) {
```

following the above comment.